cli/cmd/publish.go

```diff
@@ -18,7 +18,6 @@ import (
 
 	cloudquery_api "github.com/cloudquery/cloudquery-api-go"
 	"github.com/cloudquery/cloudquery/cli/internal/auth"
-	"github.com/cloudquery/plugin-sdk/v4/plugin"
```

Comment:
The CLI should not reference the SDK, only https://github.com/cloudquery/plugin-pb-go

Reply:
Actually I see it's used for the docs command too so we might want to refactor it at some point. The API surface the CLI should interact with is https://github.com/cloudquery/plugin-pb-go

Reply:
Yes, we should refactor this at some point. It will mean copying quite a bit of code from the SDK into plugin-pb-go I think, e.g. all the table struct definitions and how to convert from Arrow to that.